src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
```

Comment:
No reason for `Lottery` to inherit `ColorSupplier`. Looks like this is the case for [association](https://stackoverflow.com/questions/885937/what-is-the-difference-between-association-aggregation-and-composition)